ui-v2/app/router.js

```diff
@@ -67,7 +67,7 @@ export const routes = {
           _options: { path: '/health-checks' },
         },
         services: {
-          _options: { path: '/services' },
+          _options: { path: '/service-instances' },
```

Comment:
Do we mind that we are changing a URL here? 🤔

Reply:
I'll leave it as '/service-instances'